deeppavlov/dataset_readers/conll2003_reader.py

```diff
@@ -1,10 +1,11 @@
 from pathlib import Path
+from logging import getLogger
+log = getLogger(__name__)
```

Comment:
Please move `log = getLogger(__name__)` after all the imports, as [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports) states:
> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

Reply:
thanks for advice